homeassistant/components/sonos/__init__.py

```diff
@@ -208,6 +209,14 @@ def is_device_invisible(self, ip_address: str) -> bool:
 
     async def async_subscribe_to_zone_updates(self, ip_address: str) -> None:
         """Test subscriptions and create SonosSpeakers based on results."""
+        try:
+            _ = IPv4Address(ip_address)
+        except AddressValueError:
+            _LOGGER.warning(
+                "Sonos integration only supports IPv4 addresses, invalid ip_address received: %s",
+                ip_address,
+            )
+            return
```

Comment:
There a multiple ways to get into this entry point besides auto-discovering. Including sonos topology messages and direct entries in the configuration.yaml. So we'll shut those down here; rather than call SoCo and get the stack dump.

Reply:
I think we shouldn't log this. Especially in discovery, we can just ignore it.

Reply:
We can't get here from discovery. Once we connect to one sonos speaker it sends us it's network topology. If the network topology includes a V6 address we well end up at this line of code and log a warning.